mithril-explorer/src/components/PoolTicker.js

```diff
@@ -0,0 +1,45 @@
+import React, {useEffect, useState} from "react";
+import {useSelector} from "react-redux";
+import {getPool} from "../store/poolsSlice";
+import Image from "next/image";
+import {OverlayTrigger, Tooltip} from "react-bootstrap";
+
+export default function PoolTicker({aggregator, partyId, ...props}) {
+    const pool = useSelector((state) => getPool(state, aggregator, partyId));
+    const [url, setUrl] = useState(undefined);
+
+    useEffect(() => {
+        const urlWithoutNetwork = `cexplorer.io/pool/${pool.party_id}`;
+        switch (pool?.network) {
+            case 'mainnet':
+                setUrl(`https://${urlWithoutNetwork}`);
+                break;
+            case 'preprod':
+                setUrl(`https://preprod.${urlWithoutNetwork}`);
+                break;
+            case 'preview':
+                setUrl(`https://preview.${urlWithoutNetwork}`);
+                break;
+            default:
+                setUrl(undefined);
+                break;
+        }
+    }, [pool.party_id, pool.network])
+
+    return (url !== undefined)
+            ? <>
+            <a href={url} target="_blank" className="link-dark link-underline-light">
+                <OverlayTrigger overlay={<Tooltip>See in CExplorer</Tooltip>}>
+                    <span>
+                        <Image src="/explorer/cexplorer_logo.png"
+                               alt="CExplorer Logo"
+                               style={{verticalAlign:"text-top"}}
+                               width={20} height={20}/>
+                        <> </>
+                        {pool.pool_ticker}
```

Comment:
Do we display something different when we don't know the pool ticker?

Reply:
In that case the logo will be displayed alongside an link with an empty text :cold_face: , what should we do then ?

Reply:
We could simply display `N/A` with an inactive link in that case?

Reply:
We changed it so it show `Not available` in that case
![image](https://github.com/input-output-hk/mithril/assets/790521/e46e93f1-40ad-4ba1-9cb3-2e947a8719f4)